### ISSUE TITLE
[Bug]: On iOS buttons don't toggle active state properly

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/the-trans-dimension#474]
## Description

Manchester/London buttons are jittery and don't toggle properly

(github videos broken seemingly)

https://drive.google.com/drive/folders/1Mi90FjALL2pModvPWZerg2CRNTDiEjnG?usp=drive_link

Firefox video was on a poor wifi connection, Safari one isn't

## Steps to reproduce

1. View on Firefox - attempt to change pages and press buttons
2. View on Safari - attempt to change pages and press buttons

## Platform (if relevant)

iPhone 16
Mobile Firefox
Mobile Safari

**Comment by kimadactyl** [comment]
unsure why videos are not working lets try in a comment

https://github.com/user-attachments/assets/0b26e52a-e3b8-4d51-95b6-f928a7ba3687




https://github.com/user-attachments/assets/deb96d16-9fe4-4642-b17f-3c55f89d84cc

**Comment by katjam** [comment]
I don't think this is an iOS issue. I'm not sure about the loading thing - but investigating this made me realise that mobile view is not kicking in for some reason (maybe since the upgrade?)

**Comment by kimadactyl** [comment]
Still happening on iOS - if someone could check on android that'd be great
